js/src/personal/maliha/Maliha.module.css

```diff
@@ -10,7 +10,22 @@
 }
 
 .body {
-  background-color: #B07251;
-  padding: 40px;
   font-family: Sans-Serif;
-}
\ No newline at end of file
+}
+.border {
+  border-left: 1px solid #662100;
+  height: 100%;
+  position: fixed;
+  left: 10%;
+  margin-left: -3px;
```

Comment:
I'm not usually a fan of negative margins unless absolutely necessary.
Is there a way you can get the same effect by structuring things differently?